FRAuth/FRAuth/Device/HardwareCollector.swift

```diff
@@ -66,24 +65,6 @@ public class HardwareCollector: DeviceCollector {
     }
     
     
-    /// Calculates current device's total disk size in MB
-    ///
-    /// - Returns: Current device's disk size in MB
-    func getDiskSpaceSize() -> Double {
```

Comment:
Is this something used by any feature? Device Profile collector? Should we update any doc?

Reply:
no, that's ok